src/vision/calib/Calibration.cpp

```diff
@@ -0,0 +1,210 @@
+#include "MonoCalibrator.hpp"
+#include "wfov_camera_msgs/WFOVImage.h"
+#include <cv_bridge/cv_bridge.h>
+#include <sensor_msgs/image_encodings.h>
+#include <string>
+#include <vector>
+
+using namespace cv_bridge;
+
+Settings s;
+
+bool calibFinished = false;
+vector<vector<Point2f>> imagePoints;
+Mat cameraMatrix, distCoeffs;
+Size imageSize;
+int mode = CAPTURING;
+clock_t prevTimestamp = 0;
+const Scalar RED(0, 0, 255), GREEN(0, 255, 0);
+const char ESC_KEY = 27;
```

Comment:
Not that I am aware of. This is simply using the ascii code for the escape key and making it easier to read in the code.